cypress/drafts/sms_gateway/Delete_gateway.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Gateway configuratons can be deleted
+
+  Background:
+    Given the user navigated to the gateway configuration page
+
+  Scenario: The user deletes the first gateway configuration
+    Given there are some gateway configurations
+    When the user clicks on the delete button in the first row
+    Then a confirmation model should pop up
+    When the user confirms the deletion
+    Then the confirmation model should close
+    And the previously first row should not exist
+
+  Scenario: The user attempts to delete the first gateway but cancels the process
+    Given there are some gateway configurations
+    When the user clicks on the delete button in the first row
+    Then a confirmation model should pop up
+    When the user cancels the deletion
+    Then the confirmation model should close
+    And the previously first row should still exist
```

Comment:
Similar to the above comment, maybe this could be rephrased as, the first row should still exist (or something more specific)